games-arcade/performous/files/performous-1.1-boost-1.73.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/game/engine.cc b/game/engine.cc
+index 9f0f7ef..560ce71 100644
+--- a/game/engine.cc
++++ b/game/engine.cc
+@@ -8,6 +8,8 @@
+ #include <iostream>
+ #include <list>
+ 
++using namespace boost::placeholders;
```

Comment:
honestly, is full qualification a problem? `using namespace boost::placeholders;` is a really brittle fixup, and I'd prefer fully qualifying arguments and not relying on ADL at all.

Reply:
I'm not an expert; the patch is from Arch, but the Gentoo bug would have an alternative.